spec/rubocop/cop/rails/transaction_exit_statement_spec.rb

```diff
@@ -1,115 +1,111 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::Rails::TransactionExitStatement, :config do
-  it 'registers an offense when `return` is used in transactions' do
-    expect_offense(<<~RUBY)
-      ApplicationRecord.transaction do
-        return if user.active?
-        ^^^^^^ Exit statement `return` is not allowed. Use `raise` (rollback) or `next` (commit).
-      end
-    RUBY
-  end
+  shared_examples 'flags transaction exit statements' do |method|
```

Comment:
I've extracted the existing specs into `shared_examples` to make sure that all cases are covered.